docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,66 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each participant or subject: 
+
+- **CandID** also known as the **DCCID** : is a unique randomized 6-digit numeric ID (e.g. '436792').  It is completely anonymous, and is assigned automatically by Loris upon participant registration. 
+- **PSCID** (Project Study Centre ID) is configurable, can be manually entered when registering a participant, and may contain the site abbreviation, followed by sequential or randomized digits (e.g. 'MTL0006')
```

Comment:
```suggestion
- **PSCID** (Project Study Centre ID) is configurable, can be manually entered when registering a participant, and may contain the site abbreviation, followed by sequential or randomized characters (e.g. 'MTL0006')
```

The ID part can be letters or numbers so "characters" is better here.

Reply:
Yep. Sequential alpha goes `AAA`, `AAB`, `AAC` just like the numbers.

Reply:
sold